tests/test_utils/gcp_system_helpers.py

```diff
@@ -68,3 +77,92 @@ def provide_gcp_context(
     return provide_gcp_conn_and_credentials(
         key_file_path=key_file_path, scopes=scopes, project_id=project_id
     )
+
+
+class GoogleSystemTest(SystemTest):
+    @staticmethod
+    def _project_id():
+        return os.environ.get("GCP_PROJECT_ID")
+
+    @staticmethod
```

Comment:
```suggestion
    @property
```
It's a simple getter. so it should be property.

Reply:
Property cannot be accessed by classmethod. I am using classmethods because it will help in future to create pytest fixtures